### PR TITLE
Add some initial data for 1.14

### PR DESCRIPTION
**Pull request by timmyrs**
Burger data extracted using https://gist.github.com/timmyrs/7ff2fdf8c8acee895f5a7fcdfaa3cb32

### CODE REVIEW

**Comment by rom1504** [comment]
Could you use https://github.com/PrismarineJS/burger-extractor (and fix it if needed) ?

**Comment by timmyrs** [comment]
I can submit the fixes I applied to items.json to burger-extractor but otherwise I think my PHP script output the same data as burger-extractor.

**Comment by timmyrs** [comment]
Alright, I've patched up burger-extractor, ran it, and used its data, instead. However, the blocks.json it generated seems to be faulty: https://circleci.com/gh/PrismarineJS/minecraft-data/692

File: data/pc/common/protocolVersions.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "minecraftVersion": "1.14",
+    "version": 477,
+    "usesNetty": true,
+    "majorVersion": "1.14"
+  },
   {
     "minecraftVersion": "1.13.2",
```
Comment (rom1504) [inline data/pc/common/protocolVersions.json]: We need all versions in that file

**Comment by rom1504** [comment]
Let's merge as a beginning.
We will need to run the more complete generator (burger-extractor) next. Fixing it also means making it produce valid output.

**Pull request merged**